Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/InstanceSerializer.cpp

```diff
@@ -81,6 +82,15 @@ namespace AzToolsFramework
                 result.Combine(resultInstances);
             }
 
+            PrefabDomUtils::LinkIdMetadata** linkIdMetadata = context.GetMetadata().Find<PrefabDomUtils::LinkIdMetadata*>();
```

Comment:
Nitpick: (See comment below as well.) If the simplified form is used the second pointer isn't needed and this is simplified to `PrefabDomUtils::LinkIdMetadata* linkIdMetadata = context.GetMetadata().Find<PrefabDomUtils::LinkIdMetadata>(); if (linkIdMetadata) { ... }`